packages/passport-interface/src/RequestTypes.ts

```diff
@@ -589,3 +637,47 @@ export type KnownTicketTypesRequest = undefined;
  * issue the user.
  */
 export const ISSUANCE_STRING = "Issue me PCDs please.";
+
+/**
+ * Collection of tickets that some clients keep track of so that the tickets
+ * contained within it function offline.
+ */
+export interface OfflineTickets {
+  devconnectTickets: OfflineDevconnectTicket[];
+  secondPartyTickets: OfflineSecondPartyTicket[];
+}
+
+/**
+ * New empty {@link OfflineTickets}.
+ */
+export function defaultOfflineTickets(): OfflineTickets {
+  return {
+    devconnectTickets: [],
+    secondPartyTickets: []
+  };
+}
+
+/**
+ * @todo - redact this? I think devconnect tickets are fine because
+ * only event staff have access to this.
+ */
+export interface OfflineDevconnectTicket {
+  id: string;
+  attendeeEmail: string;
+  attendeeName: string;
+  eventName: string;
+  ticketName: string;
+  checkinTimestamp?: string;
+  checker: string | null;
+}
+
+/**
+ * @todo - redacting the contents of this server-side before giving it to the
+ * client probably makes sense.
+ */
```

Comment:
cc @artwyman @robknight

Reply:
As discussed, let's hash the ticket ID.

Reply:
As we discussed, I don't think I really understand the functional requirements for Zuconnect ticket scanning.  But looking at the fields, the ticket ID seems to be the only sensitive one, so Rob's hashing suggestion seems reasonable.